great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -57,7 +57,9 @@ def __init__(
             metric_domain_kwargs: used in MetricConfiguration
             metric_value_kwargs: used in MetricConfiguration
             false_positive_rate: user-configured fraction between 0 and 1 -- "FP/(FP + TN)" -- where:
-            FP stands for "false positives" and TN stands for "true negatives"; this rate specifies allowed "fall-out".
+            FP stands for "false positives" and TN stands for "true negatives"; this rate specifies allowed "fall-out"
+            (in addition, a helpful identity used in this method is: false_positive_rate = 1 - true_negative_rate).
+            round_to_nearest_integer: user-configured boolean directive for whether or not to round parameter values.
```

Comment:
Just a tiny clarifying nitpick
```suggestion
            round_to_nearest_integer: user-configured boolean directive for whether or not to round the output parameter values (min_value, max_value).
```

Reply:
🙇

Reply:
@anthonyburdi Your suggestion has been committed.  Thanks!